core/transaction/payload/Subscribe.go

```diff
@@ -0,0 +1,127 @@
+package payload
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"strings"
+
+	"github.com/nknorg/nkn/common"
+	"github.com/nknorg/nkn/common/serialization"
+	. "github.com/nknorg/nkn/errors"
+)
+
+type Subscribe struct {
```

Comment:
FYI, all data structure will be migrated to protobuf in the future, do you want to migrate later (like most txn now) or straightly start with protobuf (like sigchain)?

Reply:
I'll migrate later for consistency sake